Atomic/scan.py

```diff
@@ -123,6 +125,15 @@ def get_additional_args():
 
         self.results_dir = os.path.join(self.results, self.scanner, self.cur_time)
 
+        # Get remediation script
+        if self.args.remediate:
+            for i in self.scanners:
+                self.remediation_script = i.get("remediation_script", None)
```

Comment:
Any reason for this to be in self?  Is the remediate_script only used in this function?

Reply:
Fixed.